packages/types/src/checkin.ts

```diff
@@ -84,7 +88,7 @@ export interface MonitorConfig {
   // See: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
   timezone?: SerializedMonitorConfig['timezone'];
   // How many consecutive failed check-ins it takes to create an issue.
-  failure_issue_threshold?: number;
+  failureIssueThreshold?: SerializedMonitorConfig['failure_issue_threshold'];
   // How many consecutive OK check-ins it takes to resolve an issue.
-  recovery_threshold?: number;
```

Comment:
Although this is technically a breaking change, these values were not used for anything, and are incorrect format, so I'm fine with removing them (they were also not documented anywhere - I opened https://github.com/getsentry/sentry-docs/pull/10168 to document this PR).